### PR TITLE
change the word "blog" to an icon on the footer #1038

### PR DESCRIPTION
**Pull request by Pushkarmondal**

### CODE REVIEW

File: website/templates/includes/footer.html [hunk 1]
```diff
@@ -43,8 +43,8 @@
                   <a href="http://twitter.com/bugheist" target="_new"><i class="fa fa-twitter"></i></a>
                   <a href="http://facebook.com/bugheist" target="_new"><i class="fa fa-facebook"></i></a>
                   <a href="http://github.com/bugheist/website" target="_new"><i class="fa fa-github"></i></a>
-                  <a href="https://steemit.com/@bugheist" target="_new">blog</a>
-
+                  <!-- <a href="https://steemit.com/@bugheist" target="_new"><i class="https://user-images.githubusercontent.com/7475382/215365180-4ba57d9b-285d-448b-8261-60b9f3978eb9.svg"></i></a> -->
```
Comment (AtmegaBuzz) [inline website/templates/includes/footer.html:46]: you can remove this commented line

**Comment by fredfalcon** [comment]
Please also use the steemit svg logo not a font icon pencil.

**Comment by AtmegaBuzz** [comment]
my bad i didnt read the issue carefully and told you to add pencil icon. here's how yo can solve it 
download the image add it to the website/img/ folder rename it to blog.svg and then you can insert an image tag instead of <a> tag. 
 steemit

**Comment by fredfalcon** [comment]
It’s ok, thanks for helping out @AtmegaBuzz !

**Pull request closed without merging**